### DISCUSSION TITLE
How is the sorting made on https://searx.space?

### DISCUSSION THREAD

**Discussion by andersnordh** [searxng/searxng#3908] [Q&A] [answered]
Hi, I am just trying to understand ...

Is the public instances list to see as some kind of ranking? If so, suddenly all of us that have upgraded to 20241010... is placed in the bottom ... Could be interesting to understand what gives us a certain place :-)

Have a good one,
Anders

**Comment by return42** [comment]
> Is the public instances list to see as some kind of ranking?

No, its a more or less unordered list .. its up to the user to give one or other the preference!

**Comment by unixfox** [comment, marked as answer]
From https://github.com/searxng/searx-space/issues/99

> Currently we rank instances by TLS grade from cryptcheck, CSP grade from mozilla observatory and then the response time for a query measured by searx.space.

the version is the most important one, a newer version will be ranked way higher in the list.

**Reply by andersnordh** [reply]
Hmm, except that 2024-10-4 is ranked way higher than 2024-10-14 ... as 4 is better than 1 ... ;-)

//Anders

**Reply by unixfox** [reply]
That's very odd. It's not supposed to be like that...

Something changed because it should be 04 and not just 4

**Reply by return42** [reply]
> Something changed because it should be 04 and not just 4

Nope, the notation is 2024.9.9 or 2024.10.10 .. in the past we had some trouble with the version numbers .. git & setup.py deal them differently: https://github.com/searxng/searxng/commit/a60851bd59afe2a36e839e3b3409e9b92ab447d2
